{"uuid":"4f51b1ac-554c-4770-a2ae-4cfc05b6b5ec","historyId":"24105ff0e8d53e9cb0e2d5c049d08c7e","fullName":"BaseConfig.BaseClassTest.afterClass","labels":[{"name":"package","value":"U546.getTVSMLatestDeviceData"},{"name":"testClass","value":"U546.getTVSMLatestDeviceData"},{"name":"testMethod","value":"afterClass"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"TVS_Automation"},{"name":"subSuite","value":"U546.getTVSMLatestDeviceData"},{"name":"host","value":"meghana-Latitude-7490"},{"name":"thread","value":"111472@meghana-Latitude-7490.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"AS_ID","value":"-1"}],"links":[],"name":"afterClass","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tLatitude is not matching expected [13.010333] but found [13.00633],\n\tLongitude is not matching expected [77.55105] but found [77.53671],\n\tOdometer is not matching expected [1280] but found [3.7]","trace":"java.lang.AssertionError: The following asserts failed:\n\tLatitude is not matching expected [13.010333] but found [13.00633],\n\tLongitude is not matching expected [77.55105] but found [77.53671],\n\tOdometer is not matching expected [1280] but found [3.7]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\n\tat BaseConfig.BaseClassTest.afterClass(BaseClassTest.java:48)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:222)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:131)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\n\tat org.testng.TestNG.run(TestNG.java:1067)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1742280567559,"stop":1742280567560}